src/windows/views/files_listview.py

```diff
@@ -151,6 +151,9 @@ def add_file(self, filepath):
                 file_data["media_type"] = "image"
             elif file_data["has_audio"] and not file_data["has_video"]:
                 file_data["media_type"] = "audio"
+            else:
+                # If neither set, just assume video
+                file_data["media_type"] = "video"
```

Comment:
This exact same code is copypasted into `files_treeview.py` as well, it probably makes sense to keep them in sync. (Better would be to eliminate the redundancy, but short of that...)

Reply:
Done, pushed a commit.